src/app/GitCommands/CommitTemplateItem.cs

```diff
@@ -9,31 +9,53 @@ public sealed class CommitTemplateItem : ISerializable
         public string Name { get; set; }
         public string Text { get; set; }
         public Image? Icon { get; set; }
+        public bool Regex { get; set; }
```

Comment:
```suggestion
        public bool IsRegex { get; set; }
```

This applies universally.